CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
 
 PROJECT(CiftiLib_and_example)
 
-SET(CMAKE_CXX_FLAGS "-W -Wall")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -W -Wall")
```

Comment:
Maybe this should use CACHE, like the IGNORE_QT variable, which seems to work as expected for a configurable option?  Using the variable in its own definition looks a little disturbing, given that some invocations of SET are persistent.